File: Estrutura de Dados/edtde2/src/Merge.java
public class Merge {
    Fila fila1;
    Fila fila2;
    Fila filaFinal;

    int tamanho = fila1.dados.length + fila2.dados.length;

    int aux;

    public Merge(Fila fila1, Fila fila2) {
        this.fila1 = fila1;
        this.fila2 = fila2;
        filaFinal = new Fila(tamanho);

    }

    public void doMerge(){

    }

}

File: Prolog/merge.pl
merge(X, [], X).

merge([], X, X).

merge([X1|Y1], [X2|Y2], [X1|Z]):-
    X1 =< X2,
    merge(Y1, [X2|Y2], Z),!.

merge([X1|Y1], [X2|Y2], [X1|Z]):-
    X2 =< X1,
    merge(Y2, [X1|Y1], Z),!.


